setup.py

```diff
@@ -25,6 +25,7 @@
     'google-api-python-client>=1.6',
     'intervaltree>=2.1.0,<2.2.0',
     'pyvcf<0.7.0',
+    'mmh3>=2.5.1',
```

Comment:
Given the random breakages that we've had in the past due to pip packages getting updated, could we pin this particular version instead? Or have a lower limit of versions? I suspect it's not used by any of our dependencies.

Reply:
Thanks for clarifying.